src/components/Purpose/Accordion/ProcessData.tsx

```diff
@@ -59,8 +59,11 @@ const ProcessData = (props: { process: Process }) => {
       <DataText label={intl.purposeOfTheProcess} text={process.description} hide={!process.description} />
 
       <DataText label={intl.legalBasis} text={process.legalBases.length ? undefined : intl.legalBasisNotFound}>
-        {process.legalBases.map((legalBasis, index) =>
-          <Block key={index}><LegalBasisView legalBasis={legalBasis} /></Block>
+        {process
+          .legalBases
+          .sort((a,b)=> (a.gdpr && codelist.getShortname(ListName.GDPR_ARTICLE, a.gdpr.code)).localeCompare(b.gdpr && codelist.getShortname(ListName.GDPR_ARTICLE, b.gdpr.code)))
```

Comment:
`a.gdpr && ` Her gjør ikke noe, om gdpr er nullable får vi nullpointer her uansett. Så du må fikse det istedenfor.

Reply:
De to variabler a og b er ikke nullable. Jeg forandrer logikken.